client/my-sites/email/email-providers-in-depth-comparison/comparison-table/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { FunctionComponent } from 'react';
+import type { TranslateResult } from 'i18n-calypso';
+
+export type ProviderComparison = {
```

Comment:
I still think this name is a bit too generic, and doesn't describe what this type is for. What do you think of `EmailProviderFeatures` instead? We use a very similar naming in the `Plans` page.